Training_25/Program.cs

```diff
@@ -3,11 +3,28 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T07 branch. Assignment Q7: Pascal's Triangle.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training_25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      WriteLine ("Enter the number of rows for the Pascal's triangle:");

```

Comment:
```suggestion
      WriteLine ("Enter the number of rows for the Pascal's triangle: ");
```
Add a space after colon.